index.html

```diff
@@ -854,18 +854,14 @@ <h2 id="docconformance">
               <a>No corresponding role</a>
             </td>
             <td>
-              <p>
-                Roles:
-                <a href="#index-aria-application">`application`</a>,
-                <a href="#index-aria-document">`document`</a>,
-                <a href="#index-aria-img">`img`</a>,
```

Comment:
There are legit use cases for using frames or objects to load images. For example loading in an SVG that needs to use scripts, or that loads external fonts. I don't think this should be changed.

Reply:
again, not ready will full results.  but initial findings do make me think that under certain circumstances `img` is ok.  application and document provide no real benefit so far.  and in some cases break things.